--- th-orphans-0.8/src/Language/Haskell/TH/Instances.hs.orig	2013-12-24 07:30:29.166154606 +0100
+++ th-orphans-0.8/src/Language/Haskell/TH/Instances.hs	2013-12-24 08:07:10.199395571 +0100
@@ -1,17 +1,17 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE StandaloneDeriving, DeriveGeneric, TypeSynonymInstances, TemplateHaskell #-}
 
-{- |
-  Module      :  Language.Haskell.TH.Instances.Lift
-  Copyright   :  (c) Matt Morrow 2008
-  License     :  BSD3
-  Maintainer  :  Michael Sloan <mgsloan at gmail>
-  Stability   :  experimental
-  Portability :  portable (template-haskell)
+-- |
+--  Module      :  Language.Haskell.TH.Instances.Lift
+--  Copyright   :  (c) Matt Morrow 2008
+--  License     :  BSD3
+--  Maintainer  :  Michael Sloan <mgsloan at gmail>
+--  Stability   :  experimental
+--  Portability :  portable (template-haskell)
+--
+--  Provides Ord and Lift instances for the TH datatypes.  Also provides
+--  Show / Eq for Loc, and Ppr for Loc / Lit.
 
-  Provides Ord and Lift instances for the TH datatypes.  Also provides
-  Show / Eq for Loc, and Ppr for Loc / Lit.
--}
 module Language.Haskell.TH.Instances () where
 
 import Language.Haskell.TH.Syntax
